### DISCUSSION TITLE
[Question]:How to add custom plugins in deploy-compose.yml

### DISCUSSION THREAD

**Discussion by angpao** [danny-avila/LibreChat#4441] [Q&A]
### What is your question?

I did follow as https://www.librechat.ai/docs/development/tools_and_plugins but cannot show my custom plugin.
More information , I use deploy-compose.yml. Do you need to map volume ?

### More Details

-

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
See this: https://github.com/danny-avila/LibreChat/discussions/4465
